Misc/NEWS.d/next/C API/2021-04-22-10-46-40.bpo-43908.Co3YhZ.rst

```diff
@@ -0,0 +1,3 @@
+Introduce :const:`Py_TPFLAGS_IMMUTABLETYPE` flag for immutable type objects, and
+modify :c:func:`PyType_Ready` to set it for all static types. Patch by
```

Comment:
```suggestion
modify :c:func:`PyType_Ready` to set it to static types. Patch by
```

Reply:
Better?

```suggestion
modify :c:func:`PyType_Ready` to set it for static types. Patch by
```

Reply:
Yes, "for static types" sounds better than "for all static types".